docs/source/advanced_features/cloud_storage.rst

```diff
@@ -0,0 +1,50 @@
+##############
+Cloud Storage
+##############
+
+All data on TensorBay will be hosted on cloud.
+
+****************
+Default Storage
+****************
+| Select default storage to upload data to your private storage space on TensorBay.
+| Create a dataset with default storage:
+
+.. literalinclude:: ../../../examples/getting_started_with_tensorbay.py
+      :language: python
+      :start-after: """Create a Dataset"""
+      :end-before: """"""
+
+************************
+Authrized Cloud Storage
+************************
+
+| Tensorbay can manage your private cloud data through authorization.
+| Now TensorBay support following cloud providers:
+
+    - Aliyun OSS
+    - Amazon S3
+    - Azure Blob
+
+Config
+=======
+
+See `cloud storage instruction <https://docs.graviti.cn/guide/tensorbay/data/authorize>`_ for details about how to configure your cloud storage on TensorBay.
```

Comment:
Configuration is needed before TensorBay can access other providers' cloud. See ... 

Delete "your"